model/config.go

```diff
@@ -1091,6 +1091,7 @@ type SqlSettings struct {
 	AtRestEncryptKey            *string  `access:"environment,write_restrictable,cloud_restrictable"`
 	QueryTimeout                *int     `access:"environment,write_restrictable,cloud_restrictable"`
 	DisableDatabaseSearch       *bool    `access:"environment,write_restrictable,cloud_restrictable"`
+	ReplicaLazyReads            *bool
```

Comment:
Should this be a feature flag or a config setting? Would admins want this disabled after being determined to be stable?